### ISSUE TITLE
Inconsistencies in 3.0 spec of constraints

### ISSUE THREAD

**Issue by dosse** [sdmx-twg/sdmx-im#24]
In https://sdmx.org/wp-content/uploads/SDMX_3-0-0_SECTION_2_FINAL-1_0.pdf, starting at line 2116 the document list the different objects related to constraints, which do not seem consistent with the definitions in SDMX-ML, the KeySet is defined as:

![image](https://user-images.githubusercontent.com/1862608/202753281-9b01759f-dfaf-4036-a613-d632592bc122.png)

![image](https://user-images.githubusercontent.com/1862608/202757728-e5fe3f78-6763-4944-9d15-d2a2f00e3f10.png)

I believe that this should be clarified and made consistent.

The specs also say in line 2092: "_Note that in all cases the "operator" on the value is deemed to be "equals", unless the wildcard character is used '%'. In the latter case the "operation" is a partial matching, where the percentage character ('%') may match zero or more characters._" It is not sufficiently explicit if this only applies to CubeRegions, which are mentioned just before, or also to KeySets.

**Comment by stratosn** [comment]
Indeed, there are some leftovers from previous versions here. Thanks @dosse !
According to the IM, the `+member` should indeed move under the `DataKey` and explicitly refer to `AttributeComponent`s and `Measure`s.
For those cases, the `MemberSelection` definition covers their relationships (as also explained in the text above - lines 2066-2071).

As for the `TimeDimensionValue`, it looks like it is there because it appears in the IM as a specialisation of the `ComponentValue`, but I agree that it does not belong in the `DataKey` definition and it should be removed (also from the IM).

Finally, I also recall that the wildcard character `%` is supposed to be used only with CubeRegions, but we will have to validate this with the TWG.

**Comment by stratosn** [comment]
It is a bug in the documentation. Need to check if it affects the schemas.
